src/Serializer/Normalizer/DateTimeNormalizer4.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Serializer\Normalizer;
+
+use DateTimeInterface;
+use Symfony\Component\Serializer\Normalizer\DateTimeNormalizer as SymfonyDateTimeNormalizer;
+
+/**
+ * Extension of Symfony's DateTimeNormalize class, only for Symfony 4.
+ *
+ * @deprecated When support for Symfony 4 will be dropped, this class needs to
```

Comment:
I'd not mark this a deprecated as this class is not a deprecated class as such, but we should keep a comment that states that this is to be used only with Symfony 4.